docs/user-guide/projects/project-lists/README.md

```diff
@@ -148,6 +155,46 @@ By selecting *Overall activity* you can open a view in which all the latest glob
 
 ![overall activity meeting filter](actvity-global-filter.png)
 
+## Favorite project lists
+
+You can mark your private project lists as favorites. This includes shared, public and private project lists, but not the static ones, e.g. *Active projects* or *Archived projects*. To mark a project list as favorite click the star icon in the top right corner. 
+
+![Mark project list as favorite in OpenProject](star-project-list.png)
+
+The star will turn yellow and the favorite project list will move to the top of the list within the respective sidebar section. If multiple project lists are favored, they will be listed alphabetically.
+
+![Favorite project list in OpenProject](star-yellow-project-list.png)
+
+You can remove the star by clicking on the star icon again. 
+
+## Share project lists 
+
+You can share project lists in OpenProject, except the static project lists, e.g. *Active projects* or project lists under *Project status* section in the lefthand menu. Project lists can either be made public, i.e. shared with everyone at OpenProject or with specific users or groups.
```

Comment:
@MayaBerd lets remove the public concept also from here and keep the "shared with everyone in your instance or with specific users or groups."